governance/proposals/up/003-increase-proposal-threshold.js

```diff
@@ -0,0 +1,31 @@
+const { UPGovernor } = require('@unlock-protocol/contracts')
+
+const BASE_UP_GOVERNOR_ADDRESS = '0x65bA0624403Fc5Ca2b20479e9F626eD4D78E0aD9'
```

Comment:
That address can now be fetched from the networks package https://github.com/unlock-protocol/unlock/blob/5580b88715c348444a9a6586d28c570a15f3f4fd/packages/networks/src/networks/base.ts#L18